composer.json

```diff
@@ -9,7 +9,7 @@
     "homepage": "https://laminas.dev",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0"
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0 || ~8.5.0"
```

Comment:
Please drop support for 8.1 whilst you're here 👍

Reply:
Forgot to mention that `config.platform.php` will need to be set to `8.2.99`

Reply:
Oops, I didn't realize that was set.